### ISSUE TITLE
Staging Review finding: Page titles are identical throughout the flow

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#120700]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** govcio-vebt
**Product:** VA Form 22-10297
**Feature:** Application for High Technology Program

## Findings details
**VA.gov Experience Standard - issue:** [User encounters content with errors or inconsistencies.](https://design.va.gov/about/experience-standards/#user-encounters-accurate-consistent-content)
**VA.gov Experience Standard - category:** [Consistency](https://design.va.gov/about/experience-standards/#category-consistency)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @jasonday (Accessibility), @erinrwhite (IA)

### Description

Page titles provide context & wayfinding to all users and should be unique and match the page's purpose and include the H1's text. 

In this flow, all page titles are "Apply for VET TEC 2.0 (high-tech program) | Veterans Affairs", which doesn't provide any context or assistance to the user to help them understand the page's purpose. 

This is correct per current written guidance in VADS but new guidance for teams is currently in the works to address this issue. Filing this as a heads up to put on your backlog.

<details><summary>Link, screenshot or steps to recreate</summary>

<img width="794" height="511" alt="page title does not update" src="https://github.com/user-attachments/assets/925ea279-0f70-4fb3-876b-b0664c6c5f86" />

</details>

## Recommended action

Ensure page titles are unique for each page and follow the Content Style Guide

This is a WCAG violation, *and* VADS guidance for [title tags within form flows](https://design.va.gov/content-style-guide/title-tags#exception-2--forms-and-subtasks) still recommends keeping the same title tag/h1 on all pages in the flow. This guidance will be updated in the near future, but for now, this follows VADS guidance and is okay.

## References

- **Accessibility Defect Severity:** [3: Medium. Should be fixed in 1-3 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-3)
- **WCAG Success Criteria:** N/A
- **Modality:** N/A
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** [Title tags](https://design.va.gov/content-style-guide/title-tags)
- **Context:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://dj540s05.optimalworkshop.com/questions/f54355271969f160ef41182ba9ceb61c) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.
